build.gradle

```diff
@@ -12,7 +12,7 @@ buildscript {
 
 ext {
 	appVersion = '11.3.1' // Also update version in BuildVersion.kt
-	pluginVersion = '0.4.1' // Also update version in BuildVersion.kt
+	pluginVersion = '0.4.2' // Also update version in BuildVersion.kt
```

Comment:
For Teamscale we have 0.4.6 set as the version. Either bump version here or CR#0 it on the Teamscale master.

Reply:
Oh I completely forgot to push these changes. Version was already set to 0.4.6.